components/ListHeader/list-header.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
refactor (blocking): eslint-disable should not be use to disable all linting for files where console statements are used. Use `eslint-disable-next-line` instead. e.g.

```javascript
try {
  await navigator.clipboard.writeText(url);
  toast({ description: "Copied to clipboard", variant: "success" });
} catch (error) {
	// eslint-disable-next-line no-console
  console.log(error);
}
```

Reply:
@nickytonline please kindly check the `code standard` linters that are failing, I have worked on all the files where we have the console statement and I don't know why the code standard is still failing. Kindly let me know if there is anything am missing.
Thank you.